src/redis-helper.ts

```diff
@@ -25,6 +26,62 @@ export async function getLineupLastUpdatedDate(redisClient: redis.RedisClient,
     }
 }
 
+export async function getArtistTopTracksLastUpdated(redisClient: redis.RedisClient, artistId: string): Promise<Date> {
+    const datePromise: Promise<string> = new Promise((resolve, reject) => {
+        redisClient.hget(`artist:${artistId}`, 'top_track_ids_last_updated', (err: Error, obj: string) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(JSON.parse(obj));
```

Comment:
Should be stored as a string, not a stringified string (which turns into `"\"2022-03-24T17:20:32.051Z\""`)
```suggestion
                resolve(obj);
```